src/main/java/teammates/ui/automated/AutomatedAction.java

```diff
@@ -58,10 +59,36 @@ protected String getRequestParamValue(String paramName) {
         return HttpRequestHelper.getValueFromRequestParameterMap(request, paramName);
     }
     
+    /**
+     * Retrieves the value for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the value of the specified parameter.
+     * @throws @link{NullPostParameterException} if the parameter is not present in the http request.
```

Comment:
There is no need to `@link` here. `@throws` tag works simply by writing down the Exception name:
```java
/**
 * ...
 * @throws NullPostParameterException if ...
 */
```